samples/tab-sso/nodejs/README.md

```diff
@@ -217,10 +217,10 @@ This Javascript file is served from the `/msal-auth.js` page and handles the bro
 This Javascript file is served from the `/ssoDemo` page and handles most of the client-side authentication workflow. This file is broken into three main functions:
 
 1. getClientSideToken() -
-This function asks Teams for an authentication token from AAD. The token is displayed so you can try it in Postman.
+This function asks Teams for an authentication token from Microsoft Entra ID. The token is displayed so you can try it in Postman.
 
 2. getServerSideToken() -
-This function sends the token to the backend to exchange for elevated permissions using AAD's [on-behalf-of flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v1-oauth2-on-behalf-of-flow). In this case, it sends the token to the `/getProfileOnBehalfOf` route.
+This function sends the token to the backend to exchange for elevated permissions using Microsoft Entra's [on-behalf-of flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v1-oauth2-on-behalf-of-flow). In this case, it sends the token to the `/getProfileOnBehalfOf` route.
```

Comment:
Microsoft Entra's >> Microsoft Entra ID's

Reply:
fixed

Reply:
Not Fixed

Reply:
Thanks, fixed now.